build/build-viewer.js

```diff
@@ -140,12 +140,12 @@ async function compileJs() {
   const options = {
     output: {preamble: license}, // Insert license at top.
   };
-  const uglified = uglifyEs.minify(contents, options);
+  const uglified = Terser.minify(contents, options);
```

Comment:
you weren't kidding about it being a drop-in replacement :)

Reply:
terser is a fork of uglify so +1